tf/start.sh

```diff
@@ -17,22 +18,43 @@ fi
 
 game_num=$(cat $GAMEFILE)
 game_num=$((game_num + GAMES))
-echo "Starting with training.$game_num.gz as last game in window"
+file="training.${game_num}.gz"
+
+echo "Starting with '$file' as last game in window"
+
+train() {
+  unbuffer ./train.py --cfg=$1 --output=$2 2>&1 > "$ROOT/logs/$(date +%Y%m%d-%H%M%S).log"
+  gzip -9 $2
+  mv -v $2.gz $NETDIR
+}
+
 
 while true
 do
-  if [ -f "/home/folkert/data/run1/training.${game_num}.gz" ]
+  if [ -f "$ROOT/data/$file" ]
   then
     echo ""
-    unbuffer ./train.py --cfg=$CONFIG --output=$NET 2>&1 | tee /home2/logs/$(date +%Y%m%d-%H%M%S).log
-    TSTAMP=$(date +"%Y_%m%d_%H%M_%S_%3N")
-    FILE="$NETDIR/$TSTAMP.lc0"
-    mv -v $NET $FILE
+
+    # prepare ramdisk
+    rm -rf $RAMDISK/*
+    cp -r $ROOT/split/{train,test} $RAMDISK
+
+    # train all networks
+    for netarch in 64x6 128x10
```

Comment:
Maybe make "64x6 128x10" a variable on top of the file?

bash documentation says this is possible:
```bash
farm_hosts=(web03 web04 web05 web06 web07)

for i in ${farm_hosts[@]}; do
```